sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/aggregation/LastValueAggregationTest.java

```diff
@@ -20,13 +20,13 @@
 import java.util.Collections;
 import org.junit.jupiter.api.Test;
 
-/** Unit tests for {@link Aggregations#sum()}. */
+/** Unit tests for {@link Aggregations#lastValue()}. */
```

Comment:
I'd just delete this javadoc altogether. We've been removing them as they don't provide much value to anyone and are too easy to become out of date.

Reply:
Done.